cmd/createcluster.go

```diff
@@ -81,9 +83,11 @@ func newCreateClusterCmd(runFunc func(io.Writer, clusterConfig) error) *cobra.Co
 }
 
 func bindClusterFlags(flags *pflag.FlagSet, config *clusterConfig) {
+	flags.StringVar(&config.Name, "name", clusterName, "Optional cosmetic cluster name")
```

Comment:
Suggest not defaulting to "", suggest using current default that is `clusterName=local`.

Reply:
Also, technically not optional if there is a default, suggest removing "Optional" from description.

Reply:
Or if empty cluster name is fine, then at least remove `clusterName` const